Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -1128,8 +1128,7 @@ Vector<String> AccessibilityRenderObject::determineDropEffects() const
     // Order is aria-dropeffect, dropzone, webkitdropzone
     const AtomString& dropEffects = getAttribute(aria_dropeffectAttr);
     if (!dropEffects.isEmpty()) {
-        String dropEffectsString = dropEffects.string();
-        dropEffectsString.replace('\n', ' ');
+        String dropEffectsString = makeStringByReplacingAll(dropEffects.string(), '\n', ' ');
         return dropEffectsString.split(' ');
```

Comment:
I don’t think we need the local here. Should just do this as a one-liner.

Reply:
Ok.